browser/src/control/jsdialog/Widget.Edit.ts

```diff
@@ -104,14 +104,17 @@ class EditWidget {
 		if (this.callback) callbackToUse = this.callback;
 		if (typeof callbackToUse === 'function')
 			callbackToUse(this.edit.input.value);
-		else
+		else {
+			var eventType = 'change';
+			if (e.key === 'Enter') eventType = 'activate'; // this is probably breaking some stuff, just use to test TODO: think of something better
```

Comment:
I see you need activate event to call: [SearchFindEntryActivateHandler](https://opengrok.libreoffice.org/s?refs=SearchFindEntryActivateHandler&project=core)

It looks correct, please implement "activate" case for "edit" control in `vcl/jsdialog/executor.cxx` in core
Remove comment.

Reply:
case here: https://gerrit.libreoffice.org/c/core/+/187729